### ISSUE TITLE
clear api does not work with waitForEvents option

### ISSUE THREAD

**Issue by Debashis9012** [getgauge/taiko#529]
### Expected behavior
clear api should work properly with the option ``` waitForEvents ```
### Actual behavior
It throwing the below error:
```
> clear({waitForEvents:['DOMContentLoaded', 'loadEventFired', 'networkAlmostIdle', 'networkIdle', 'firstPaint', 'firstContentfulPaint', 'firstMeaningfulPaint']})
 ✘ Error: undefined not found, run `.trace` for more info.
```
### Steps to reproduce
1.  openBrowser()
2. goto("google.com")
3. write("TestAutomation")
4. clear({waitForEvents:['DOMContentLoaded', 'loadEventFired', 'networkAlmostIdle', 'networkIdle', 'firstPaint', 'firstContentfulPaint', 'firstMeaningfulPaint']})

```
commit: 9a876b8
```

**Comment by Apoorva-GA** [comment]
Fix should be available in commit 7ae7368

**Comment by UrielCh** [comment]
Hi,
I think that there some error in the doc:

`options.waitForEvents` string Events available to wait for ['DOMContentLoaded', 'loadEventFired', 'networkAlmostIdle', 'networkIdle', 'firstPaint', 'firstContentfulPaint', 'firstMeaningfulPaint']] (optional, default ['firstMeaningfulPaint'])


should be replaced by

`options.waitForEvents` string[] Events available to wait for ['DOMContentLoaded', 'loadEventFired', 'networkAlmostIdle', 'networkIdle', 'firstPaint', 'firstContentfulPaint', 'firstMeaningfulPaint']] (optional, default ['firstMeaningfulPaint'])

**Comment by Apoorva-GA** [comment]
@UrielCh thanks for pointing it out. Fixed in commit c206a75

**Comment by Debashis9012** [comment]
Both issue and documentation has been verified and found fixed.
